x-pack/test/apm_api_integration/tests/service_groups/wait_for_active_alert.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ToolingLog } from '@kbn/tooling-log';
+import expect from '@kbn/expect';
+import { Client } from '@elastic/elasticsearch';
+
+const WAIT_FOR_STATUS_INCREMENT = 1000;
+
+export async function waitForActiveAlert({
+  ruleId,
+  waitMillis = 10000,
+  esClient,
+  log,
+}: {
+  ruleId: string;
+  waitMillis?: number;
+  esClient: Client;
+  log: ToolingLog;
+}): Promise<Record<string, any>> {
+  if (waitMillis < 0) {
+    expect().fail(`waiting for active alert for rule ${ruleId} timed out`);
+  }
+
+  const searchParams = {
+    index: '.alerts-observability.apm.alerts-*',
+    size: 1,
+    query: {
+      bool: {
+        filter: [
+          {
+            term: {
+              'kibana.alert.rule.producer': 'apm',
+            },
+          },
+          {
+            term: {
+              'kibana.alert.status': 'active',
+            },
+          },
+          {
+            term: {
+              'kibana.alert.rule.uuid': ruleId,
+            },
+          },
+        ],
+      },
+    },
+  };
+  const response = await esClient.search(searchParams);
```

Comment:
Can we use the APM alerts client (`getApmAlertsClient`)?

Reply:
No, I don't think it's an option in within the tests